software/webdb.yml

```
@@ -0,0 +1,14 @@
+name: WebDB
+website_url: https://webdb.app
+source_code_url: https://gitlab.com/web-db/app
+description: Open Source and Efficient Database IDE.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+tags:
+  - Database Management
+  - Software Development - IDE & Tools
+  - Self-hosting Solutions
```

I dont think this fits this tag as its more for all in one solutions.

```suggestion
  - Database Management
  - Software Development - IDE & Tools
```